### PR TITLE
fix(DB/creature): delete duplicate Big Roy

### PR DESCRIPTION
**Pull request by Quickson**
Deleted a duplicate NPC Big Roy

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3676

## TESTS PERFORMED:
Tested on Win10 without any problems

## HOW TO TEST THE CHANGES:
1) .go c 203496
2) Big Roy there must be one

## KNOWN ISSUES AND TODO LIST:
- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1606162431492361200.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1606162431492361200');
+DELETE FROM creature WHERE guid = 203340;
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1606162431492361200.sql:2]: Please add backticks around column names

Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1606162431492361200.sql:2]:
```suggestion
DELETE FROM `creature` WHERE `guid` = 203340;
```

**Review by LukasVolgger — DISMISSED** [review]

**Comment by LukasVolgger** [comment]
2020-11-23 20:32:41 Creature (GUID: 203340) does not exist but has a record in `creature_addon`

File: data/sql/updates/pending_db_world/rev_1606162431492361200.sql (hunk 1 above)
Comment (Quickson) [reply data/sql/updates/pending_db_world/rev_1606162431492361200.sql]: @LukasVolgger fixed, thanks!

**Comment by Quickson** [comment]
@LukasVolgger yes, I did. Initially only in the game. But now I also re-deployed the database and deleted and waypoint duplicate creature

File: data/sql/updates/pending_db_world/rev_1606162431492361200.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1606162431492361200');
+DELETE `creature`, `creature_addon`, `waypoint_data`
+FROM
+    `creature`, `creature_addon`, `waypoint_data`
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1606162431492361200.sql:4]: i have never seen a delete query like that, u sure it works?

Comment (Quickson) [reply data/sql/updates/pending_db_world/rev_1606162431492361200.sql:4]: @BarbzYHOOL i tested this query on my MySQL 8.0.22. Example request i looked at https://stackoverflow.com/a/28621990

Comment (LukasVolgger) [reply data/sql/updates/pending_db_world/rev_1606162431492361200.sql:4]: @Quickson looks good to me so we don't need to delete from every single table!

**Review by LukasVolgger — APPROVED** [review]

**Comment by Quickson** [comment]
Thanks for approving @LukasVolgger :)

**Comment by LukasVolgger** [comment]
I'll test tonight and if everything is okay we will merge this! @Quickson

**Comment by LukasVolgger** [comment]
Tested, working as expected! Big Roy is spawned by a core script so the related DB entries are invalid and deleted correctly with your SQL query. 

Good work @Quickson!

image

**Comment by Quickson** [comment]
> Tested, working as expected! Big Roy is spawned by a core script so the related DB entries are invalid and deleted correctly with your SQL query. 
> 
> Good work @Quickson!
> 
> image
> 

Thanks @LukasVolgger!)

**Review by locus313 — APPROVED** [review]

**Comment by locus313** [comment]
nice work @Quickson . Like the single sql delete statement to remove the records in all 3 tables :)

**Comment by Quickson** [comment]
Thanks for approving @locus313 :)

**Review by Si1ker — APPROVED** [review]

**Pull request merged**